src/main.cpp

```diff
@@ -693,6 +693,34 @@ void setup()
     // Now that the mesh service is created, create any modules
     setupModules();
 
+#ifdef LED_PIN
+    // Turn LED off after boot, if disabled by config
```

Comment:
Note that this is a separate issue relating to `heartbeat_disabled`, independent of this PR's scope.